UPGRADE.md

```diff
@@ -0,0 +1,114 @@
+# Upgrade to Ceramic One from Kubo
+
+This document describes how to upgrade an existing Ceramic node using Kubo to run with `ceramic-one` while preserving existing data.
+
+## Overview
+
+Follow these steps for basic upgrade of a Ceramic node with some downtime.
+
+1. Shutdown existing `js-ceramic` node.
+2. Migrate data from IPFS blocks into Sqlite database file `ceramic-one` can consume.
+3. Start `ceramic-one` daemon.
+4. Configure `js-ceramic` to run against `ceramic-one`.
+5. Start `js-ceramic` node and validate.
+
+If you require a zero downtime upgrade see the section below that expands the basic steps for a near zero downtime upgrade.
+
+### Shutdown Existing Node
+
+First shutdown both the `js-ceramic` and `kubo` processes using whatever mechanism your infrastructure dictates.
+
+### Migrate IPFS Blocks to Ceramic Events
+
+Kubo stores data as IPFS blocks on disk.
+If you are not using the disk backed Kubo storage first migrate data to an on disk store following Kubo's instructions.
+
+These IPFS blocks are the raw pieces of Ceramic Events.
+In order to preserve this data we need to migrate them into Ceramic Events stored in as a Sqlite database file.
+
+The `ceramic-one` command contains a subcommand for migrating data from IPFS blocks.
+Install the `ceramic-one` binary following the instructions in README.md.
+
+See the complete usage information for the migration command:
+
+    ceramic-one migrations from-ipfs --help
+
+The command requires an input directory where the IPFS blocks are found and where the output storage directory.
+
+The following command will migrate data assuming the default directories for both Kubo and `ceramic-one`.
+Adjust the command arguments as needed by your infrastructure.
+
+    ceramic-one migrations from-ipfs -i ~/.ipfs/blocks -o ~/.ceramic-one --network mainnet
+
+This command will migrate all blocks stored in the `~/.ipfs/blocks` directory into the database file `~/.ceramic-one/db.sqlite3` for the `mainnet` network.
+As output from the command you should see a total number of events migrated, this will not be the same as the number of blocks as multiple blocks construct into a single event.
+You will also see a number of errors. This represents the number of blocks found that are part of a Ceramic Event but for whatever reasons could not be constructed into a complete event.
+Any error will be logged for investigations.
+Any blocks stored in Kubo that are not Ceramic Events will be ignored.
+
+### Start `ceramic-one` Daemon
+
+Once the data is migrated its time to start the node again.
+The `ceramic-one` process is started in place of the `kubo` process, Kubo is no longer needed.
+
+Read the full usage of the `ceramic-one` daemon:
+
+    ceramic-one daemon --help
+
+Once you are familiar with configuring the daemon start it with defaults or specify any arguments specific to your infrastructure.
+
+    ceramic-one daemon
+
+You should see logs indicating the daemon is running.
```

Comment:
Being more specific of what output the devs should see here would be ideal to make it easier to verify that devs are following the upgrade process correctly